codespell_lib/data/dictionary.txt

```diff
@@ -14388,6 +14389,7 @@ mnay->many
 mocule->module
 mocules->modules
 modell->model
+modelling->modeling
```

Comment:
`modelling` is the british spelling. https://www.grammar.com/modelling_vs._modeling